packages/vscode/.env_sample

```diff
@@ -1,2 +1,2 @@
-PRODUCTION=<1> # Default to dry-run, to actually publish, use PRODUCTION=1
+ENVIRONMENT=<1> # Default to dry-run, to actually publish, use ENVIRONMENT=PRODUCTION
```

Comment:
Is <1> still correct here?

Reply:
anything besides the actual value `PRODUCTION` will be fine